samples/kfp-user-guide/README.md

```diff
@@ -343,3 +343,34 @@ When developing a Kubeflow pipeline for the Tekton backend, please be aware that
 Therefore, we have prohibited the kfp-tekton compiler from putting artifacts and parameter output files in the container's root directory. We recommend placing the output files inside a new directory under root to avoid this problem, such as `/tmp/`. The [condition](/samples/flip-coin/condition.py) example shows how the output files can be stored. Alternatively, you can learn [how to create reusable components](https://www.kubeflow.org/docs/pipelines/sdk/component-development/) in a component.yaml where you can avoid hard coding your output file path.
 
 This also applies to the Argo backend with k8sapi and kubelet executors, and it's the recommended way to avoid the [race condition for Argo's PNS executor](https://github.com/argoproj/argo/issues/1256#issuecomment-494319015).
+
+## Migration from Argo backend
+
+### Argo variables
+
+Variables like `{{workflow.uid}}` are currently not supported. See [this PR](https://github.com/kubeflow/kfp-tekton/pull/321).
+
+### Absolute paths in commands
+
+Absolute paths in component commands are required, e.g. `python app/run.py` works in Argo, in Tekton it has to be `python /app/run.py`.
```

Comment:
Actually, there's a way to enable relative paths. It's documented in the prerequisites of the [samples README.md](https://github.com/kubeflow/kfp-tekton/tree/master/samples#prerequisites) and [dev setup](https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/README.md#tekton-cluster).

Also, we changed all these for the users if they are deploying the whole Kubeflow. We probably need to move the dev setup instructions over here in case someone is creating a custom kfp-tekton deployment.